front-end/src/components/Admin/Grid/Filter/index.scss

```diff
@@ -19,22 +19,22 @@
     width: 150px;
     height: 30px;
     margin-right: 20px;
-    background-color: $white;
+    background-color: $configBtnColorTwo;
     border: none;
     border-radius: 3px;
   }
 
   button {
     margin-right: 20px;
     padding: 6px 5px;
-    color: $pink-bg;
-    background-color: $white;
-    border: 1px solid $pink-bg;
+    color: $configBtnColorOne;
+    background-color: $configBtnColorTwo;
```

Comment:
$configBtnColorOne and $configBtnColorTwo don't sound like semantic colors to me :)